src/transport/PeerConnectionState.h

```diff
@@ -77,8 +77,11 @@ class PeerConnectionState
     uint64_t GetLastActivityTimeMs() const { return mLastActivityTimeMs; }
     void SetLastActivityTimeMs(uint64_t value) { mLastActivityTimeMs = value; }
 
-    SecureSession & GetSecureSession() { return mSecureSession; }
-    const SecureSession & GetSecureSession() const { return mSecureSession; }
+    SecureSession & GetSenderSecureSession() { return mSenderSecureSession; }
+    const SecureSession & GetSenderSecureSession() const { return mSenderSecureSession; }
+
+    SecureSession & GetReceiverSecureSession() { return mReceiverSecureSession; }
+    const SecureSession & GetReceiverSecureSession() const { return mReceiverSecureSession; }
```

Comment:
I guess we still need these so we can do the `DeriveSecureSession` thing?  Maybe just leave the non-const ones?

I'd still like to figure out some way to not have consumers have to think about these getters at all... maybe a followup to figure that out?

Reply:
Removed the const getters.